docs/operator-manual/notifications/services/email.md

```diff
@@ -6,9 +6,9 @@ The Email notification service sends email notifications using SMTP protocol and
 
 * `host` - the SMTP server host name
 * `port` - the SMTP server port
-* `username` - username
-* `password` - password
-* `from` - from email address
+* `username` - user email address 
+* `password` - user app password  // To create your app password, follow this link https://myaccount.google.com/apppasswords
```

Comment:
These are "gmail" specific and should be moved to their own section. 

After the parameters section, I suggested adding "Using Gmail" and specify the username needs to be an email address and the app password should be created using this link. 

Not all SMTP configurations require using email address for example.

Reply:
I agree 100% with @todaywasawesome . This should also state that this is specific when using private (non Google Workspace) GMail accounts (IIRC the process is slightly different for those cases)

Reply:
sure thing!